storage/client/src/main/java/com/azure/storage/blob/AppendBlobAsyncClient.java

```diff
@@ -92,21 +94,15 @@ public static AppendBlobClientBuilder appendBlobClientBuilder() {
      *         {@link Metadata}
      * @param accessConditions
      *         {@link BlobAccessConditions}
-     * @param context
-     *         {@code Context} offers a means of passing arbitrary data (key/value pairs) to an
-     *         {@link com.azure.core.http.HttpPipeline}'s policy objects. Most applications do not need to pass
-     *         arbitrary data to the pipeline and can pass {@code Context.NONE} or {@code null}. Each context object is
-     *         immutable. The {@code withContext} with data method creates a new {@code Context} object that refers to
-     *         its parent, forming a linked list.
      *
      * @return
      *      A reactive response containing the information of the created appended blob.
      */
-    public Mono<AppendBlobCreateHeaders> create(BlobHTTPHeaders headers, Metadata metadata,
-                BlobAccessConditions accessConditions, Context context) {
+    public Mono<Response<AppendBlobItem>> create(BlobHTTPHeaders headers, Metadata metadata,
```

Comment:
create returns way more information that appedBlock and yet they both return the same type?

Reply:
This is written based on https://gist.github.com/jaschrep-msft/59f5d4312d0ef555f5ed454c9a504a13. All the important headers returned in the raw create method are included in AppendBlobItem class.